pytket/docs/circuit_class.rst

```diff
@@ -20,7 +20,375 @@ following keyword arguments are supported:
 ``condition``, or using the pair ``condition_bits`` and ``condition_value`` to
 condition on a specified set of bit values.)
 
-.. currentmodule:: pytket._tket.circuit.Circuit
-.. autoclass:: pytket._tket.circuit.Circuit
-   :special-members:
-   :members:
+.. currentmodule:: pytket.circuit.Circuit
+.. autoclass:: pytket.circuit.Circuit
+
+   .. automethod:: __init__
+
+   .. automethod:: __eq__
+   
+   .. automethod:: __iter__
+
+   .. automethod:: __mul__
+
+   .. automethod:: __repr__
+
+   .. automethod:: __rshift__
+
+   .. automethod:: __str__
+
+   .. automethod:: __getstate__
+
+   .. automethod:: __setstate__
+
+   .. automethod:: add_gate
+
+   .. automethod:: append
+
+   .. automethod:: add_circuit
+
+   .. automethod:: add_qubit
+
+   .. automethod:: add_bit
+
+   .. automethod:: add_phase
+
+   .. autoproperty:: name
+
+   .. autoproperty:: n_qubits
+
+   .. autoproperty:: n_bits
+
+   .. autoproperty:: phase
+
+   .. autoproperty:: qubits
+
+   .. autoproperty:: bits
+
+   .. autoproperty:: n_gates
+
+   .. autoproperty:: is_symbolic
+
+   .. automethod:: add_q_register
+
+   .. automethod:: get_q_register
+
+   .. automethod:: add_c_register
+
+   .. automethod:: get_c_register
+
+   .. autoproperty:: q_registers
+
+   .. autoproperty:: c_registers
+
+   .. automethod:: rename_units
+
+   .. automethod:: add_blank_wires
+
+   .. automethod:: remove_blank_wires
+
+   .. automethod:: flatten_registers
+
+   .. autoproperty:: qubit_readout
+
+   .. autoproperty:: bit_readout
+
+   .. autoproperty:: opgroups
+
+   .. autoproperty:: is_simple
+
+   .. autoproperty:: qubit_to_bit_map
+
+   .. automethod:: commands_of_type
+
+   .. automethod:: ops_of_type
+
+   .. automethod:: n_gates_of_type
+
+   .. automethod:: n_1qb_gates
+
+   .. automethod:: n_2qb_gates
+
+   .. automethod:: n_nqb_gates
+
+   .. automethod:: free_symbols
+
+   .. automethod:: symbol_substitution
+
+   .. automethod:: substitute_named
+
+   .. automethod:: depth
+
+   .. automethod:: depth_2q
+
+   .. automethod:: depth_by_type 
```

Comment:
There's a space at the end of this line: that's why the script wasn't detecting it.

Reply:
thanks, fixed it